readme.md

```diff
@@ -127,7 +123,36 @@ Deep changes will not trigger the callback. Only changes to the immediate proper
 Type: `Function`<br>
 Default: [Object.is](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is) ([SameValue](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#Same-value_equality) equality)
 
-The function receives two arguments, the two values to be compared for equality. True should be returned if the two values are determined to be equal.
+The function receives two arguments, the two values to be compared for equality. True should be returned if the two values are determined to be equal. 
+
+##### ignoreSymbols
+
+Type: `boolean`<br>
+Default: false
+
+Setting properties as Symbols won't trigger the callback.
+
+
+### onChange.target(object)
+
+Returns the original unwatched object.
+
+#### object
+
+Type: `object`
+
+Object that is already being watched for changes.
+
+
+### onChange.unsubscribe(object)
```

Comment:
Don't forget to keep the TypeScript definition file and the docs in sync. These are missing from index.d.ts.

Reply:
Sorry, I published 1.6.0 right before you typed this. I'll get that and your last commit in a patch. Also, using np, it failed trying to push to github but I don't see any documentation on the np readme about signing in to git.

Reply:
It's assumed that you've already set up Git. Or am I misunderstanding your question? How did it fail? Error?

Reply:
Interesting. I use GitKraken, so I probably never set up git on this device. I'll deal with that, but it would be nice if np didn't fail in that scenario. I'll post an issue about it in that repo.

Reply:
Can I tell np to only do certain steps? It never created the release draft.

Reply:
No, but for every step before the publish, if an error happens, it will revert the changes. But obviously not possible after publish. Would be useful to be able to just generate the release draft though. You should open an issue on `np`.